### PR TITLE
Test protocol builder #519

### PR DESCRIPTION
**Pull request by cullerton**
- Changes for working with UVA PB urls
  - Protocol Builder model
  - Study Info script
  - Study Service _is_valid_study() 
- Added logging.
- Added switch in config/default for using UVA urls or PB Mock urls.
- Fixed tests
  - mocked protocol builder data
  - required_docs, study_details

### CODE REVIEW

File: config/logging.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+    'urllib3.connectionpool': {
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+    'elasticsearch': {
```
Comment (danfunk) [inline config/logging.py:20]: These elastic loggers aren't relevant to this project.

File: config/logging.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      'propagate': False
+    },
+  },
+  "formatters": {
+    "simple": {
+      "format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    }
+  },
+  "root": {
+    "level": "DEBUG",
```
Comment (danfunk) [inline config/logging.py:37]: Debug might me WAY too loud and create a bunch of junk.

File: crc/__init__.py [hunk 3]
```diff
@@ -18,8 +17,8 @@
 from apscheduler.schedulers.background import BackgroundScheduler
 from werkzeug.middleware.proxy_fix import ProxyFix
 
-
-logging.basicConfig(level=logging.INFO)
+from config.logging import logging_config
```
Comment (danfunk) [inline crc/__init__.py:20]: Turn this off for testing

**Review by danfunk — CHANGES_REQUESTED** [review]
Just super minor stuff, overall great to see all of this.

File: config/logging.py (hunk 2 above)
Comment (cullerton) [reply config/logging.py:37]: I set console to DEBUG and file to INFO

**Pull request merged**